packages/react/package.json

```diff
@@ -62,7 +62,7 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
-    "@web3-onboard/core": "^2.16.0",
+    "@web3-onboard/core": "^2.16.1-alpha.2",
```

Comment:
We will need to bump the version of this package as well

Reply:
good catch! thx!!